server/game/gameSystems/SelectCardSystem.ts

```diff
@@ -36,6 +36,7 @@ export interface ISelectCardProperties<TContext extends AbilityContext = Ability
  * Functions the same as a targetResolver and used in situations where one can't be created (e.g., costs).
  */
 export class SelectCardSystem<TContext extends AbilityContext = AbilityContext> extends CardTargetSystem<TContext, ISelectCardProperties<TContext>> {
+    public override readonly name: string = 'select-card';
```

Comment:
nit, styling: we've been using camelCase for descriptor strings generally (the card names we use for tests are kind of a weird exception)